src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Controllers/ApprenticeController.cs

```diff
@@ -97,9 +96,27 @@ public async Task<IActionResult> ChangeStartDate(ChangeStartDateRequest request)
         [HttpGet]
         [Route("{apprenticeshipHashedId}/change-employer/price", Name = RouteNames.ApprenticeNewPrice)]
         [DasAuthorize(CommitmentOperation.AccessApprenticeship, ProviderFeature.ChangeOfEmployer)]
-        public Task<IActionResult> NewPrice(ChangePriceRequest request)
+        public async Task<IActionResult> ChangePrice(ChangePriceRequest request)
```

Comment:
We should call this action `Price` to match the route - and rename all associated objects (request, viewmodel, view, etc).